src/website/layouts/partials/stencilDoc.pug

```diff
@@ -0,0 +1,137 @@
+              if prop.name !== 'extraClass'
+                tr
+                  td
+                    div
+                      code=prop.name
+                      if prop.required
+                        strong (required)
+                  td
+                    div
+                      if prop.attr
+                        code=prop.attr
+                      else
+                        | --
+                  td
+                    div
+                      if prop.deprecation !== undefined
+                        .-text--danger **[DEPRECATED]** #{prop.deprecation}
+                        br
+                        br
+                      =prop.docs
+                  td
+                    div
+                      code=prop.type
+                  td
+                    div
+                      if prop.default
+                        code=prop.default
+                      else
+                        em undefined
+
+mixin events(events)
+  if events.length
+    h3 Events
+    section.a-table.a-table__options.-bordered.-my--3
+      div
+        table.-text(cellpadding='0', cellspacing='0')
+          thead
+            tr
+              th
+                div Event
+              th
+                div Description
+              th
+                div Type
+          tbody
+            each ev in events
+              tr
+                td
+                  div
+                    code=ev.event
+                td
+                  div=ev.docs
+                td
+                  div
+                    code=`CustomEvent<${ev.detail}>`
+
+mixin methods(methods)
+  if methods.length
+    h3 Methods
+
+    each method in methods
+      h4=method.signature
+      if method.docs
+        p.-text=method.docs
+      if method.parameters.length
+        h5 Parameters
+        if method.parameters.length
+          section.a-table.a-table__options.-bordered.-my--3
+            div
+              table.-text(cellpadding='0', cellspacing='0')
+                thead
+                  tr
+                    th
+                      div Name
+                    th
+                      div Type
+                    th
+                      div Description
+                tbody
+                  each parameter in method.parameters
+                    tr
+                      td
+                        div
+                          code=parameter.name
+                      td
+                        div
+                          code=parameter.type
+                      td
+                        div=parameter.docs
+      if method.returns
+        h5 Returns
+        p.-text Type: #{method.returns.type}
+        p.-text= method.returns.docs
+
+//mixin slots(slots)
+  // TODO
+
+//mixin styles(styles)
+  // TODO
+
+//mixin deps(config, cmp, cmps)
+  // TODO
+
+each component in data.components
+  if component.tag === name
+    h2 &lt;#{component.tag} /&gt; documentation
```

Comment:
@dani-cl-madrid looks like even this is redundant as it will be used in a pug file with specific title which will be representing our CE. Thoughts?

Reply:
@srinath-koneru I found a problem on the date component. It has two custom element chi-date and chi-date-picker. So I went this way to solve it. I have two more different solutions:

1. Title is not rendered by the plugin so we it has to be set from the outside. 
2. Title is rendered by the plugin but there is set by an attribute. There might exist a default value for the title. 

Which one do you prefer?

Reply:
@dani-cl-madrid I would prefer 1.

Reply:
Roger that. @srinath-koneru Some default title?

Reply:
@dani-cl-madrid Arrghh I cannot think of any synonyms that can hold (Props, Events, Methods and all), looks like back to square one. @SergioQCL can you think of any?

Reply:
@srinath-koneru @SergioQCL How about **Detailed documentation** or **Documentation details**

Reply:
+1 for documentation details

Reply:
@dani-cl-madrid @SergioQCL How about **Additional Details**.

Reply:
@srinath-koneru mmm, in my opinion it seems more accurate to use documentation instead of additional.

Reply:
@SergioQCL me too but isn't the html itself represents a documentation, calling documentation details again kind of feels redundant.

Reply:
@srinath-koneru yes, it makes sense. Additional could do, but I'll add a little suggestion here: additional usage details.